docs/source/guides/staking_guide.rst

```diff
@@ -134,37 +192,86 @@ Interactive Method
 
     | ~ | Staker | Worker | # | Value    | Duration     | Enactment
     |   | ------ | ------ | - | -------- | ------------ | -----------------------------------------
-    | 0 | 0xbb01 | 0xbb02 | 0 | 15000 NU | 41 periods . | Aug 04 12:15:16 CEST - Sep 13 12:15:16 CEST
-    | 1 | 0xbb01 | 0xbb02 | 1 | 15000 NU | 30 periods . | Aug 20 12:15:16 CEST - Sep 18 12:15:16 CEST
-    | 2 | 0xbb03 |    -   | 0 | 30000 NU | 30 periods . | Aug 09 12:15:16 CEST - Sep 9 12:15:16 CEST
+    | 0 | 0xbb01 | 0xdead | 0 | 15000 NU | 41 periods . | Aug 04 12:15:16 CEST - Sep 13 12:15:16 CEST
+    | 1 | 0xbb02 | 0xbeef | 1 | 15000 NU | 30 periods . | Aug 20 12:15:16 CEST - Sep 18 12:15:16 CEST
+    | 2 | 0xbb03 | 0x0000 | 0 | 30000 NU | 30 periods . | Aug 09 12:15:16 CEST - Sep 9 12:15:16 CEST
+
+If the Worker in the list is shown as ``0x0000``, it means that you didn't yet
+attach any node to your Staker, so you still have to do it!
 
 
-*Bond an Ursula to a Staker*
+Bond an Ursula to a Staker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After you created an Ethereum node for your worker (see below about the worker),
+you can set the worker. The stake which doesn't yet have a worker will be
+highlighted in yellow:
 
 .. code:: bash
 
-    (nucypher)$ nucypher stake set-worker
+    (nucypher)$ nucypher stake set-worker --hw-wallet
 
     ======================================= Active Stakes =========================================
 
     | ~ | Staker | Worker | # | Value    | Duration     | Enactment
     |   | ------ | ------ | - | -------- | ------------ | -----------------------------------------
-    | 0 | 0xbb01 | 0xbb02 | 0 | 15000 NU | 41 periods . | Aug 04 12:15:16 CEST - Sep 13 12:15:16 CEST
-    | 1 | 0xbb01 | 0xbb02 | 1 | 15000 NU | 30 periods . | Aug 20 12:15:16 CEST - Sep 18 12:15:16 CEST
-    | 2 | 0xbb03 |   -    | 0 | 30000 NU | 39 periods . | Aug 09 12:15:16 CEST - Sep 16 12:15:16 CEST
+    | 0 | 0xbb01 | 0xdead | 0 | 15000 NU | 41 periods . | Aug 04 12:15:16 CEST - Sep 13 12:15:16 CEST
+    | 1 | 0xbb02 | 0xbeef | 1 | 15000 NU | 30 periods . | Aug 20 12:15:16 CEST - Sep 18 12:15:16 CEST
+    | 2 | 0xbb03 | 0x0000 | 0 | 30000 NU | 30 periods . | Aug 09 12:15:16 CEST - Sep 9 12:15:16 CEST
 
     Select Stake: 2
-    Enter Worker Address: 0xbb04c4fE50f91eF73c5dD6eD89f38D55A6b1EdCA
+    Enter Worker Address: 0xbeefc4fE50f91eF73c5dD6eD89f38D55A6b1EdCA
     Worker 0xbb04c4fE50f91eF73c5dD6eD89f38D55A6b1EdCA successfully bonded to staker 0xbb03...
 
     OK!
 
+Please note that the address should be in the format where checksum is encoded
+in the address. However, geth shows addresses in the lower case. You can convert
```

Comment:
Is it easier for us to just accept lower case addresses rather than asking people to manually convert by copy/paste?

Reply:
I think so! We can do the same as we did for Felix